lib/elixir/lib/inspect/algebra.ex

```diff
@@ -324,18 +324,37 @@ defmodule Inspect.Algebra do
 
   ## Examples
 
-      iex> doc = Inspect.Algebra.container_doc("[", Enum.to_list(1..5), "]",
-      ...>         %Inspect.Opts{limit: :infinity}, fn i, _opts -> to_string(i) end)
+      iex> doc =
+      ...>   Inspect.Algebra.container_doc(
+      ...>     "[",
+      ...>     Enum.to_list(1..5),
+      ...>     "]",
+      ...>     %Inspect.Opts{limit: :infinity},
+      ...>     fn i, _opts -> to_string(i) end
+      ...>   )
```

Comment:
Can we break this one into variables instead?

Reply:
Sure, I'll update it in next few minutes